### PR TITLE
Open Handler test for MsgCreate/Edit Oracle script 

### LINKED ISSUE #1697: Handler test for MsgCreate/Edit Oracle script
Same as #1696

### PR DESCRIPTION
**Pull request by Benzbeeb**
Fixed #1697

### CODE REVIEW

File: chain/x/oracle/keeper/oracle_script.go [hunk 1]
```diff
@@ -60,6 +60,21 @@ func (k Keeper) EditOracleScript(ctx sdk.Context, id types.OracleScriptID, new t
 	return nil
 }
 
+// EditOracleScript edits the given oracle script by id and flushes it to the storage. Panic if not exists.
+func (k Keeper) MustEditOracleScript(ctx sdk.Context, id types.OracleScriptID, new types.OracleScript) {
+	oracleScript, err := k.GetOracleScript(ctx, id)
```
Comment (sorawit) [inline chain/x/oracle/keeper/oracle_script.go]: Let's remove `EditOracleScript` since it's not used anymore.

**Review by sorawit — CHANGES_REQUESTED** [review]

File: chain/x/oracle/keeper/oracle_script.go [hunk 2]
```diff
@@ -60,6 +60,21 @@ func (k Keeper) EditOracleScript(ctx sdk.Context, id types.OracleScriptID, new t
 	return nil
 }
 
+// EditOracleScript edits the given oracle script by id and flushes it to the storage. Panic if not exists.
```
Comment (taobun) [inline chain/x/oracle/keeper/oracle_script.go]: Fix comment to match this function behavior

File: chain/x/oracle/handler_test.go [hunk 3]
```diff
@@ -167,58 +155,220 @@ func TestEditDataSourceFail(t *testing.T) {
...
+	defer deleteFile(code)
 
-// 	newOwner := sdk.AccAddress([]byte("anotherowner"))
-// 	newName := "oracle_script_2"
-// 	newDescription := "description_2"
-// 	newCode := []byte("code_2")
-// 	sender := sdk.AccAddress([]byte("owner"))
-// 	schema := "schema"
-// 	sourceCodeURL := "sourceCodeURL"
+	require.Nil(t, err)
```
Comment (taobun) [inline chain/x/oracle/handler_test.go]: require.NoError

File: chain/x/oracle/handler_test.go [hunk 4]
```diff
@@ -167,58 +155,220 @@ func TestEditDataSourceFail(t *testing.T) {
...
-// 	soureCodeURL := "sourceCodeURL"
+	msg := types.NewMsgCreateOracleScript(Owner.Address, name, description, gzippedCode, schema, url, Alice.Address)
+	_, err := oracle.NewHandler(keeper)(ctx, msg)
+	defer deleteFile(code)
 
-// 	msg := types.NewMsgEditOracleScript(1, newOwner, newName, newDescription, newCode, schema, soureCodeURL, sender)
-// 	_, err = handleMsgEditOracleScript(ctx, keeper, msg)
-// 	require.NotNil(t, err)
-// }
+	require.Nil(t, err)
```
Comment (taobun) [inline chain/x/oracle/handler_test.go]: ditto

**Review by taobun — CHANGES_REQUESTED** [review]

**Review by taobun — DISMISSED** [review]

File: chain/x/oracle/keeper/oracle_script.go [hunk 5]
```diff
@@ -44,11 +44,11 @@ func (k Keeper) AddOracleScript(ctx sdk.Context, oracleScript types.OracleScript
 	return id
 }
 
-// EditOracleScript edits the given oracle script by id and flushes it to the storage.
-func (k Keeper) EditOracleScript(ctx sdk.Context, id types.OracleScriptID, new types.OracleScript) error {
+// MustEditOracleScript edits the given oracle script by id and flushes it to the storage. Panic if not exists.
+func (k Keeper) MustEditOracleScript(ctx sdk.Context, id types.OracleScriptID, new types.OracleScript) {
 	oracleScript, err := k.GetOracleScript(ctx, id)
```
Comment (sorawit) [inline chain/x/oracle/keeper/oracle_script.go]: MustGet

**Review by sorawit — CHANGES_REQUESTED** [review]

**Review by taobun — APPROVED** [review]

**Review by sorawit — APPROVED** [review]

**Pull request merged**